package.json

```diff
@@ -34,7 +34,7 @@
     "typescript-transform-paths": "^3.3.1"
   },
   "dependencies": {
-    "@caravinci/arranger-server": "^2.15.4",
+    "@caravinci/arranger-server": "^2.15.5",
```

Comment:
Arranger's 'ping' path can now be customised. Responds with a better message than 'ok'